code/create_spoken_forms.py

```diff
@@ -1,8 +1,9 @@
 import itertools
 import re
 from collections import defaultdict
+from collections.abc import Mapping
```

Comment:
What's up with these changes from typing to collections.abc?

Reply:
good question. I think my branch may have been out of sync and not up to date. I can put it back.

Reply:
Our autoformatter started doing that last week.  Apparently as of 3.9 they're starting [to deprecate](https://peps.python.org/pep-0585/#:~:text=Importing%20those%20from%20typing%20is%20deprecated) the versions from `typing` in favour of those from `collections.abc`

Reply:
Yeah, I think this is coming from a new [pyupgrade](https://github.com/asottile/pyupgrade) release.

Reply:
Looks like this change was made in #921, so if you rebase this change should disappear.